src/Rules/Methods/MissingMethodReturnTypehintRule.php

```diff
@@ -31,7 +34,11 @@ public function processNode(Node $node, Scope $scope): array
 			throw new \PHPStan\ShouldNotHappenException();
 		}
 
-		$methodReflection = $scope->getClassReflection()->getNativeMethod($node->name->name);
+		if (!$scope->isInTrait()) {
```

Comment:
I had to leave this in to get https://github.com/phpstan/phpstan-strict-rules/blob/a1b5b6e0e6f876ba5649c42f2092ba015f641bdb/tests/Rules/Methods/MissingMethodReturnTypehintRuleTest.php#L48-L63 to work. Would you not expect to check the trait directly?

Reply:
Please follow my advice I gave here: https://github.com/phpstan/phpstan-strict-rules/pull/70#issuecomment-461736870

Reply:
I can't get that to work... I end up with the using class name but the line number from the trait (eg `42: Method MissingMethodReturnTypehint\Foo::c1() has no return typehint specified.`).

Reply:
Can you push this change as a new commit so I can review it?

Reply:
Pushed 10eee51.

Reply:
The results look fine to me, just the declaring class is wrong there - which is something we might fix in PHPStan's core, but it's not a big deal.